confgenerator/testdata/invalid/linux/metrics-receiver_sqlserver_non_windows/golden_error

```diff
@@ -0,0 +1 @@
+metrics receiver with type "sqlserver" is not supported. Supported metrics receiver types: [activemq, apache, cassandra, couchdb, elasticsearch, hadoop, hbase, hostmetrics, jvm, kafka, memcached, mongodb, mysql, nginx, postgresql, rabbitmq, redis, solr, tomcat, wildfly, zookeeper].
```

Comment:
Other tests of this kind are named `metrics-receiver_invalid_type_*` (e.g., `metrics-receiver_invalid_type_mssql`). For consistency, this should be named `metrics-receiver_invalid_type_sqlserver`.

I can see us renaming these wholesale to `metrics-receiver_sqlserver_unsupported` (`metrics-receiver_iis_unsupported`, `metrics-receiver_mssql_unsupported`, `logging-receiver_windows_event_log_unsupported`) to ensure consistent test prefix, but I'd strongly suggest doing that in a separate PR, and making this one consistent with the existing tests.

Reply:
fixed